test/fixtures/tape-standalone.js

```diff
@@ -10,19 +10,23 @@ if (navigator.userAgent.includes("Chrome/")) {
 }
 
 // Export as a global a wrapped test function which enforces a timeout by default.
-// eslint-disable-next-line
 /**
  * @param {string} desc
- * @param {import("tape").TestOptions|null} [options]
- * @param {(test: import("tape").Test) => void|Promise<void>} fn
+ *        The test description
+ * @param {object} [options]
+ *        The test options, can be omitted.
+ * @param {number} [options.timeout=DEFAULT_TIMEOUT]
+ *        The time after which the test fails automatically, unless it has already passed.
+ * @param {boolean} [options.skip]
+ *        Whether the test case should be skipped.
+ * @param {function(tape.Test):void|Promise<void>} fn
```

Comment:
Remove the return type. `void|Promise<void>` is ambiguous (as the one can also read it as `fn = function or Promise`). To cover the return type, you can change the description to  `The (optionally async) function that runs the test case`.

Reply:
I ended up putting the&nbsp;return type inside a&nbsp;parenthesis to&nbsp;make it&nbsp;clearer (even though it&nbsp;doesn’t matter).